### PR TITLE
 Update current documentation to reflect current release pratice

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/cut-sw-release.md [hunk 1]
```diff
@@ -368,36 +372,38 @@ cd docker-osg-wn
...
 2.  The release manager emails the announcement to `vdt-discuss@opensciencegrid.org`
 3.  The release manager asks the GOC to distribute the announcement by [opening a ticket](https://ticket.opensciencegrid.org/goc/other)
 4.  The release manager closes the tickets marked 'Ready for Release' in the release's JIRA filter using the 'bulk change' function.
     Also set the Fix Versions field to the appropriate value(s) and uncheck the box that reads "Send mail for this update"
 
+Day 3: Update the ITB
+---------------------
+
+Now that the release has had a chance to propogate to all the mirrors, update the Madison ITB site by following
+the [yum update section](../infrastructure/madison-itb/#doing-yum-updates) of the Madison ITB document.
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: site-relative links please https://opensciencegrid.github.io/technology/documentation/style-guide/#links

File: docs/release/cut-sw-release.md [hunk 2]
```diff
@@ -368,36 +372,38 @@ cd docker-osg-wn
...
+Remember, it may be advisable to stop the HTCondor daemons according to the [HTCondor pre-release testing instructions](itb-testing/#installing-htcondor-prerelease).
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: Let's be clearer here; tell them when they "may" want to stop the condor daemons and which hosts they need to stop them on. Or you can just tell them to do it every time.

File: docs/release/cut-data-release.md [hunk 3]
```diff
@@ -61,7 +67,7 @@ Day 2: Pushing the Release
 
 ### Step 1: Push from pre-release to release
 
-This script moves the packages into release, clones releases into new version-specific release repos, locks the repos and regenerates them. Afterwards, it produces `*release-note*` files that should be used to update the release note pages. Clone it from the github repo and run the script:
+This script moves the packages into release, clones releases into new version-specific release repos, locks the repos and regenerates them. Afterward, it produces `*release-note*` files that should be used to update the release note pages. Clone it from the github repo and run the script:
```
Comment (brianhlin) [inline docs/release/cut-data-release.md]: This is fine but seems unnecessary? http://grammarist.com/usage/afterward-afterwards/

File: docs/release/cut-sw-release.md [hunk 4]
```diff
@@ -141,7 +141,11 @@ To test pre-release, you will be kicking off a manual VM universe test run from
 !!! note
     If there are failures, consult the release-manager before proceeding.
 
-### Step 3: Regenerate the build repositories
+### Step 3: Test Pre-Release on the Madison ITB site
+
+Test the pre-release on the Madison ITB by following the [ITB pre-release testing instructions](itb-testing/).
```
Comment (brianhlin) [inline docs/release/cut-sw-release.md]: What about people that don't have access to the Madison ITB?

Comment (timtheisen) [reply docs/release/cut-sw-release.md]: I think that everyone that can release will have access to the Madison ITB. Two ways to accomplish this. 1. Only Madison folks release. 2. Give access to remote team members.

**Comment by timtheisen** [comment]
I still have to make changes to the data release section regarding the ITB.

**Pull request merged**